configtx/orderer.go

```diff
@@ -62,6 +62,16 @@ type OrdererOrg struct {
 	name     string
 }
 
+// EtcdRaftOptionsValue wraps the EtcdRaftOptions value.
```

Comment:
This is actually kind of a hack, it's not actually wrapping an EtcdRaftOptions value since there is no config value by that name, rather it wraps the configurability of the etcdraft options in the `ConsensusType`. Maybe something like 
```
// EtcdRaftOptionsValue encapsulates the configuration functions used to modify an etcdraft configuration's options.